robot-core/src/test/resources/simple.diff

```diff
@@ -1,4 +1,4 @@
 0 axioms in Ontology 1 but not in Ontology 2:
 
 1 axioms in Ontology 2 but not in Ontology 1:
-+ SubClassOf(<https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test1>[Test 1] owl:Thing)
++ SubClassOf(<https://github.com/ontodev/robot/robot-core/src/test/resources/simple.owl#test1>[Test 1] <owl:Thing>)
```

Comment:
It's a very minor thing, but when using labels, `owl:Thing` is enclosed in the angle brackets. When not using labels, it isn't enclosed. I wish there were a way to make the simple output a little easier to read but... this might be as good as we're going to get.

Reply:
I think it is more consistent this way. For the label output, we have an identifier enclosed in angle brackets, possibly followed by a label in square brackets. To me it is confusing if sometimes the identifier is not in angle brackets.